src/base/preferences.cpp

```diff
@@ -1054,6 +1056,115 @@ void Preferences::setExecutionLogEnabled(bool b)
     setValue("Preferences/ExecutionLog/enabled", b);
 }
 
+bool Preferences::executionLogNormalType() const
```

Comment:
Sure. Can you recommend a type? I am thinking `char` and setting its first 4bits to on/off. (needs extra code though)